dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/LoggerUtils.java

```diff
@@ -67,20 +67,15 @@ private LoggerUtils() {
      * build job id
      *
      * @param affix Task Logger's prefix
-     * @param processDefId process define id
      * @param processInstId process instance id
      * @param taskId task id
      * @return task id format
      */
     public static String buildTaskId(String affix,
-                                     int processDefId,
                                      int processInstId,
                                      int taskId) {
         // - [taskAppId=TASK_79_4084_15210]
-        return String.format(" - %s%s-%s-%s-%s]", TASK_APPID_LOG_FORMAT, affix,
-                processDefId,
-                processInstId,
-                taskId);
+        return String.format(" - %s%s-%s-%s]", TASK_APPID_LOG_FORMAT, affix, processInstId, taskId);
```

Comment:
It is suggested to modify it to be consistent with the log path rule.
It is convenient to debug through log.

Reply:
This is taskAppId, not log path